apps/judicial-system/backend/src/app/modules/notification/notification.config.ts

```diff
@@ -0,0 +1,39 @@
+import { defineConfig } from '@island.is/nest/config'
+
+export const notificationModuleConfig = defineConfig({
+  name: 'NotificationModule',
+  load: (env) => ({
+    production: env.optional('NODE_ENV') === 'production',
+    email: {
+      fromEmail: env.required('EMAIL_FROM', 'ben10@omnitrix.is'),
```

Comment:
I am just curious. I see that this email is currently being used in the env config. What email is this?

Reply:
This is something devops added a long time ago, some "valid" dummy email that the build system does not complain about. Real email addresses are not allowed in the repository.